client/post-editor/editor-sidebar/feedback-header.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import EditorPostType from 'post-editor/editor-post-type';
+
+// TODO: Consider merging this with EditorSidebarHeader depending on final design
+export function FeedbackSidebarHeader( { translate, closeFeedback } ) {
+	return (
+		<div className="editor-sidebar__header">
+			<EditorPostType isSettings />
+			<Gridicon icon="arrow-right" />
+			{ translate( 'Share Draft' ) }
+			<Button
+				compact
+				borderless
+				className="editor-sidebar__back"
+				onClick={ closeFeedback }
+				title={ translate( 'Close feedback' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **99** times:
`translate( 'Feedback' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).